frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -89,53 +80,42 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
 
   return (
     <>
-      <Modal.Header>
-        <Modal.CloseButton onClick={closeOverlay} />
-        <CatalogItemHeader
-          iconClass={iconClass}
-          iconImg={imgUrl}
-          title={name}
-          vendor={`${version} provided by ${provider}`}
-        />
-      </Modal.Header>
-      <Modal.Body>
-        <div className="modal-body-content">
-          <div className="modal-body-inner-shadow-covers">
-            <div className="co-catalog-page__overlay-body">
-              <PropertiesSidePanel>
-                {!installed ? (
-                  <Link
-                    className="pf-c-button pf-m-primary co-catalog-page__overlay-create"
-                    to={createLink}
-                  >
-                    Install
-                  </Link>
-                ) : (
-                  <Button
-                    className="co-catalog-page__overlay-create"
-                    isDisabled={!installed}
-                    onClick={() => history.push(uninstallLink())}
-                    variant="secondary"
-                  >
-                    Uninstall
-                  </Button>
-                )}
-                <PropertyItem label="Operator Version" value={version || notAvailable} />
-                <PropertyItem label="Provider Type" value={providerType || notAvailable} />
-                <PropertyItem label="Provider" value={provider || notAvailable} />
-                <PropertyItem label="Repository" value={repository || notAvailable} />
-                <PropertyItem label="Container Image" value={containerImage || notAvailable} />
-                <PropertyItem label="Created At" value={createdAt || notAvailable} />
-                <PropertyItem label="Support" value={support || notAvailable} />
-              </PropertiesSidePanel>
-              <div className="co-catalog-page__overlay-description">
-                {getHintBlock()}
-                {longDescription ? <MarkdownView content={longDescription} /> : description}
-              </div>
+      <div className="modal-body-content">
+        <div className="modal-body-inner-shadow-covers">
```

Comment:
The shadows are no longer rendering when scrolling is required.
Before:
![Screen Shot 2019-12-12 at 9 13 05 AM](https://user-images.githubusercontent.com/895728/70719337-cfde3900-1cbf-11ea-860c-acaaeabc04c8.png)
After:
![Screen Shot 2019-12-12 at 9 15 08 AM](https://user-images.githubusercontent.com/895728/70719398-eab0ad80-1cbf-11ea-8fa7-d65540f8707f.png)

Reply:
@rebeccaalpert I can take a look at what's needed to get the scroll shadows working, since i've worked on implementing those in the past.